src/bucket/BucketManager.cpp

```diff
@@ -1202,6 +1202,8 @@ BucketManager::resolveBackgroundEvictionScan(
         evictionCandidates = mEvictionFuture.get();
     }
 
+    auto timer = mBucketListEvictionMetrics.time.TimeScope();
```

Comment:
Are we missing a metric for the first part of eviction (the one that runs in the background).